cli/command/init/common.go

```diff
@@ -2,6 +2,9 @@ package init
 
 import (
 	"fmt"
+	"github.com/saucelabs/saucectl/internal/cypress"
```

Comment:
Why don't we follow a practice when inside imports standard libraries go first, after that external packages and finally our own packages?

Reply:
`gofmt` doesn't truly care. We can certainly enforce `goimports` if the team finds it useful.

Reply:
We can move this conversation to the tech hudle.